plugins/modules/dcnm_policy.py

```diff
@@ -438,7 +442,9 @@ def dcnm_policy_validate_input(self):
 
         policy_spec = dict(
             name=dict(required=True, type="str"),
-            create_additional_policy=dict(required=False, type="bool", default=True),
+            create_additional_policy=dict(
```

Comment:
Looks like there are quite a few formatting only changes in this diff set.  I assume this was from running `ansible-test` ?

Reply:
yes